### ISSUE TITLE
Folder 'All Mail' not found

### ISSUE THREAD

**Issue by DenBond7** [FlowCrypt/flowcrypt-android#642]
```
[SUMMARY]
java.lang.IllegalArgumentException: Folder 'All Mail' not found
com.flowcrypt.email.util.exception.ManualHandledException: Handled manually:

[STACK_TRACE]
com.flowcrypt.email.util.exception.ManualHandledException: Handled manually:
        at com.flowcrypt.email.util.exception.ExceptionUtil$Companion.handleError(ExceptionUtil.kt:30)
        at com.flowcrypt.email.ui.activity.MessageDetailsActivity.onArchiveMsgClicked(MessageDetailsActivity.kt:263)
        at com.flowcrypt.email.ui.activity.fragment.MessageDetailsFragment.runMsgAction(MessageDetailsFragment.kt:445)
        at com.flowcrypt.email.ui.activity.fragment.MessageDetailsFragment.onOptionsItemSelected(MessageDetailsFragment.kt:197)
        at androidx.fragment.app.Fragment.performOptionsItemSelected(Fragment.java:2587)
        at androidx.fragment.app.FragmentManagerImpl.dispatchOptionsItemSelected(FragmentManager.java:3368)
        at androidx.fragment.app.FragmentController.dispatchOptionsItemSelected(FragmentController.java:356)
        at androidx.fragment.app.FragmentActivity.onMenuItemSelected(FragmentActivity.java:442)
        at androidx.appcompat.app.AppCompatActivity.onMenuItemSelected(AppCompatActivity.java:196)
        at androidx.appcompat.view.WindowCallbackWrapper.onMenuItemSelected(WindowCallbackWrapper.java:109)
        at androidx.appcompat.view.WindowCallbackWrapper.onMenuItemSelected(WindowCallbackWrapper.java:109)
        at androidx.appcompat.app.ToolbarActionBar$2.onMenuItemClick(ToolbarActionBar.java:64)
        at androidx.appcompat.widget.Toolbar$1.onMenuItemClick(Toolbar.java:204)
        at androidx.appcompat.widget.ActionMenuView$MenuBuilderCallback.onMenuItemSelected(ActionMenuView.java:781)
        at androidx.appcompat.view.menu.MenuBuilder.dispatchMenuItemSelected(MenuBuilder.java:840)
        at androidx.appcompat.view.menu.MenuItemImpl.invoke(MenuItemImpl.java:158)
        at androidx.appcompat.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:991)
        at androidx.appcompat.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:981)
        at androidx.appcompat.widget.ActionMenuView.invokeItem(ActionMenuView.java:625)
        at androidx.appcompat.view.menu.ActionMenuItemView.onClick(ActionMenuItemView.java:151)
        at android.view.View.performClick(View.java:6605)
        at android.view.View.performClickInternal(View.java:6582)
        at android.view.View.access$3100(View.java:778)
        at android.view.View$PerformClick.run(View.java:25897)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6762)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
Caused by: java.lang.IllegalArgumentException: Folder 'All Mail' not found
        ... 30 more
java.lang.IllegalArgumentException: Folder 'All Mail' not found
        at com.flowcrypt.email.ui.activity.MessageDetailsActivity.onArchiveMsgClicked(MessageDetailsActivity.kt:263)
        at com.flowcrypt.email.ui.activity.fragment.MessageDetailsFragment.runMsgAction(MessageDetailsFragment.kt:445)
        at com.flowcrypt.email.ui.activity.fragment.MessageDetailsFragment.onOptionsItemSelected(MessageDetailsFragment.kt:197)
        at androidx.fragment.app.Fragment.performOptionsItemSelected(Fragment.java:2587)
        at androidx.fragment.app.FragmentManagerImpl.dispatchOptionsItemSelected(FragmentManager.java:3368)
        at androidx.fragment.app.FragmentController.dispatchOptionsItemSelected(FragmentController.java:356)
        at androidx.fragment.app.FragmentActivity.onMenuItemSelected(FragmentActivity.java:442)
        at androidx.appcompat.app.AppCompatActivity.onMenuItemSelected(AppCompatActivity.java:196)
        at androidx.appcompat.view.WindowCallbackWrapper.onMenuItemSelected(WindowCallbackWrapper.java:109)
        at androidx.appcompat.view.WindowCallbackWrapper.onMenuItemSelected(WindowCallbackWrapper.java:109)
        at androidx.appcompat.app.ToolbarActionBar$2.onMenuItemClick(ToolbarActionBar.java:64)
        at androidx.appcompat.widget.Toolbar$1.onMenuItemClick(Toolbar.java:204)
        at androidx.appcompat.widget.ActionMenuView$MenuBuilderCallback.onMenuItemSelected(ActionMenuView.java:781)
        at androidx.appcompat.view.menu.MenuBuilder.dispatchMenuItemSelected(MenuBuilder.java:840)
        at androidx.appcompat.view.menu.MenuItemImpl.invoke(MenuItemImpl.java:158)
        at androidx.appcompat.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:991)
        at androidx.appcompat.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:981)
        at androidx.appcompat.widget.ActionMenuView.invokeItem(ActionMenuView.java:625)
        at androidx.appcompat.view.menu.ActionMenuItemView.onClick(ActionMenuItemView.java:151)
        at android.view.View.performClick(View.java:6605)
        at android.view.View.performClickInternal(View.java:6582)
        at android.view.View.access$3100(View.java:778)
        at android.view.View$PerformClick.run(View.java:25897)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6762)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
```

ref https://mail.google.com/mail/u/0/#search/%22All+Mail'+not+found%22

**Comment by DenBond7** [comment]
I'm moving this issue to `later` (still no more info after last debug code)

**Comment by DenBond7** [comment]
Fixed via #718
